.gitignore

```diff
@@ -23,6 +23,9 @@ build/
 [Bb]in/
 [Oo]bj/
 
+# Enable scenarios in integration tests
+!integrationtests/scenarios/*/before/**
```

Comment:
is this a good thing!?

Reply:
As far as i understand, idea is - all files in "test-scenario/before" folder is "setup" for test.
I think it's slightly easier to remove .idea, .dotsettings etc from commit than add ignored dlls/pdbs.